feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/fee/utils/HydraDxQuoteSharedComputation.kt

```diff
@@ -79,9 +85,17 @@ class HydraDxQuoteSharedComputation(
         val key = "HydraDxAssetConversion:${accountId.toHexString()}"
 
         return computationalCache.useCache(key, scope) {
-            val storageSharedRequestBuilder = storageSharedRequestsBuilderFactory.create(chain.id)
+            val subscriptionBuilder = storageSharedRequestsBuilderFactory.create(chain.id)
             val assetConversion = assetConversionFactory.create(chain)
-            assetConversion.runSubscriptions(accountId, storageSharedRequestBuilder)
+
+            scope.launch {
```

Comment:
As I mentined in the previous PR review, you should use aggregate scope from the CompututationalCache and not the scope received in the arguments. Otherwise if the first scope that was used to initiate cache gets cancelled despite some other scopes still being present - subscriptions will be cancelled as well